daly-bms-uart.h

```diff
@@ -20,8 +19,121 @@ class Daly_BMS_UART
         DISCHARGE_CHARGE_MOS_STATUS = 0x93,
         STATUS_INFO = 0x94,
         CELL_VOLTAGES = 0x95,
-        FAILURE_CODES = 0x98
+        FAILURE_CODES = 0x98,
+        DISCHRG_FET = 0xD9, //Switching off charge and discharge
```

Comment:
I don't see these commands in the protocol document, how did you find these command values?

Reply:
i pulled it out from the communication with the pc software